endpointmanager/cmd/jsonexport/main.go

```diff
@@ -29,6 +30,12 @@ func main() {
 	ctx := context.Background()
 	log.Info("Successfully connected to DB!")
 
+	var emptyJSON []byte
+	if _, err := os.Stat("/etc/lantern/exportfolder/fhir_endpoints_fields.json"); os.IsNotExist(err) {
+		err = ioutil.WriteFile("/etc/lantern/exportfolder/fhir_endpoints_fields.json", emptyJSON, 0644)
+		helpers.FailOnError("Failed to create empty JSON export file", err)
+	}
```

Comment:
I don't think we want this here, since it creates an empty json file if one doesn't exist, which is important if you haven't run the jsonexport yet and you click the download JSON button on the shiny dashboard. I don't think it necessarily matters where it's run, it could either go back to the capabilityquerier or maybe in sendendpoints if it needs to stay in the endpointmanager.